inventory/environ.py

```diff
@@ -266,15 +268,15 @@ def convert_path_windows_to_nix(filepath):
         return filepath
 
 def getUFSplunkVariables(vars_scope):
-    if os.environ.get('SPLUNK_DEPLOYMENT_SERVER', False):
-        vars_scope["splunk"]["deployment_server"] = os.environ.get('SPLUNK_DEPLOYMENT_SERVER')
-    if os.environ.get('SPLUNK_ADD', False):
-        vars_scope["splunk"]["add"] = os.environ.get('SPLUNK_ADD').split(',')
-    if os.environ.get('SPLUNK_BEFORE_START_CMD', False):
-        vars_scope["splunk"]["before_start_cmd"] = os.environ.get('SPLUNK_BEFORE_START_CMD').split(',')
-    if os.environ.get('SPLUNK_CMD', False):
-        vars_scope["splunk"]["cmd"] = os.environ.get('SPLUNK_CMD').split(',')
-    vars_scope["docker_version"] = '18.06.0'
```

Comment:
I removed `docker_version` because it does not seem like we're using this anywhere.